alohomora/req.py

```diff
@@ -231,23 +240,17 @@ def login_one_factor(self, username, password):
         elif password not in payload.values():
             alohomora.die("Couldn't find right form field for password!")
 
-        # Some IdPs don't explicitly set a form action, but if one is set we should
-        # build the idpauthformsubmiturl by combining the scheme and hostname
-        # from the entry url with the form action target
-        # If the action tag doesn't exist, we just stick with the
-        # idpauthformsubmiturl above
         for inputtag in soup.find_all(re.compile('form', re.IGNORECASE)):
             action = inputtag.get('action')
             if action:
                 parsedurl = urlparse.urlparse(self.idp_url)
-                idpauthformsubmiturl = parsedurl.scheme + "://" + parsedurl.netloc + action
+                idpauthformsubmiturl = parsedurl.scheme + "://" + parsedurl.netloc + "/dag/module.php/core/loginuserpass.php"
```

Comment:
The form action they converted to "?", hence changing the action part to the loginuserpass endpoint.